### ISSUE TITLE
[Compatibility] Please add Ruins compatibility

### ISSUE THREAD

**Issue by DevTimur** [TeamMetallurgy/Atum2#183]
Mod called [Ruins (Structure Spawning System)](https://www.curseforge.com/minecraft/mc-mods/ruins-structure-spawning-system), please add ability to spawn it in Atum dimension.
Or maybe add own structure spawning system, so I can add own buildings. If that's easier. Thanks!

**Comment by sscards55** [comment]
Did you change any of the .tml structure files in ruins_config to spawn in the atum dimension?. Structure only spawn in vanilla worlds by default.

**Comment by DevTimur** [comment]
Sure. I read many suggestions.
I tryed adding "Dimension ID 17" in Ruins.cfg
I tryed adding "Dimension ID atum" in Ruins.cfg
I tryed adding "dimensionsToSpawnIn=atum" in *.tml and "dimensionsToSpawnIn=17", I tryed adding tml in "sand_dunes", "sand_hills" and "sand_plains" with and without dimension specification.
I think I tryed every combination of this options.

Maybe i'm doing somthing wrong, maybe you try it?
